firebase/build.boot

```diff
@@ -17,11 +16,13 @@
 
 (deftask package []
   (comp
-    (download :url (str "https://github.com/firebase/firebase-bower/archive/v" +lib-version+ ".zip")
-              :checksum "559003EA8EF0375A16140BBB8AB87905"
-              :unzip true)
-    (sift :move {#"firebase-bower-([\d\.]*)/firebase.js" "cljsjs/development/firebase.inc.js"})
-    (sift :include #{#"^cljsjs"})
-    (deps-cljs :name "cljsjs.firebase")
-    (pom)
-    (jar)))
+   (download :url (str "http://registry.npmjs.org/firebase/-/firebase-" +lib-version+ ".tgz")
+             ;;:checksum "29ab75208d45f96e312121d1f546fceec77653de"
```

Comment:
I agree, but it fails...
Is a shasum different from a checksum?
```
dist: {
shasum: "29ab75208d45f96e312121d1f546fceec77653de",
tarball: "https://registry.npmjs.org/firebase/-/firebase-3.2.0.tgz"
},
```
I'm thinking of removing the comment to avoid confusion